lib/apr/views/bidding_slack_view.ex

```diff
@@ -4,6 +4,10 @@ defmodule Apr.Views.BiddingSlackView do
   import Apr.Views.Helper
 
   def render(event) do
```

Comment:
is it possible now to have `bidding_slack_view` being called without subscription?  I think your method on line 10 should be enough.